docs/queues.rst

```diff
@@ -3,5 +3,112 @@
 
 .. versionadded:: 4.2
 
+.. testsetup::
+
+    from tornado import ioloop, gen, queues
+    io_loop = ioloop.IOLoop.current()
+
 .. automodule:: tornado.queues
+
+   Classes
+   -------
+
+   Queue
+   ^^^^^
+   .. autoclass:: Queue
     :members:
+
+    .. testcode::
```

Comment:
How does this interact with the autoclass-generated docs? Should these examples go into the class docstring instead? (I generally prefer to keep things in the docstrings to avoid this question and to have one source of truth for a given document)